quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/ImportDistTest.java

```diff
@@ -42,6 +42,6 @@ void testImport(KeycloakDistribution dist) {
         cliResult.assertMessage("Realm 'master' imported");
         cliResult.assertMessage("Import finished successfully");
         cliResult.assertNoMessage("Changes detected in configuration");
-        cliResult.assertNoMessage("Listening on:");
+        cliResult.assertNoMessage("Listening on: http");
```

Comment:
Change on Quarkus side, as the `Listening on:` is present, but no URL is specified, so no HTTP connections should not be accepted.